src/scripts/main.js

```diff
@@ -1,3 +1,228 @@
+  table.append(el);
+}
+
+// Sort table
+tHead.addEventListener('click', (evt) => {
+  const title = evt.target;
+  const idx = title.cellIndex;
+
+  if (checkColumn === idx && asc) {
+    return descSort(tBody.rows, idx).forEach(rowEl => {
+      appendElement(tBody, rowEl);
+    });
+  }
+
+  return ascSort(tBody.rows, idx).forEach(rowEl => {
+    appendElement(tBody, rowEl);
+    checkColumn = idx;
+  });
+});
+
+// Add class active to rows
+tBody.addEventListener('click', (evt) => {
+  const targetEl = evt.target;
+
+  Array.from(tBody.rows).forEach(row => row.classList.contains('active')
+    ? row.classList.remove('active')
+    : targetEl.parentNode.classList.add('active'));
+});
+
+// Add Form on page
+const form = document.createElement('form');
+
+form.classList.add('new-employee-form');
+form.setAttribute('method', 'get');
+
+form.insertAdjacentHTML('afterbegin', `
+    <label for="name">
+      Name:
+      <input
+        id="name"
+        type="text"
+        name="name"
+        data-qa="name"
+      >
+    </label>
+
+    <label for="position">
+      Position:
+      <input
+        id="position"
+        name="position"
+        type="text"
+        data-qa="position"
+      >
+    </label>
+
+    <label for="office">
+      Office:
+      <select
+        id="office"
+        name="office"
+        data-qa="office"
+      >
+        <option value="Tokyo">Tokyo</option>
+        <option value="Singapore">Singapore</option>
+        <option value="London">London</option>
+        <option value="New York">New York</option>
+        <option value="Edinburgh">Edinburgh</option>
+        <option value="San Francisco">San Francisco</option>
+      </select>
+    </label>
+
+    <label for="age">
+      Age:
+      <input
+        id="age"
+        type="number"
+        name="age"
+        data-qa="age"
+      >
+    </label>
+
+    <label for="salary">
+      Salary:
+      <input
+        id="salary"
+        type="number"
+        name="salary"
+        data-qa="salary"
+      >
+    </label>
+
+    <button id="submit" type="submit">Save to table</button>
+`);
+document.body.append(form);
+
+// Add data from form in table
+function onAddRow(e) {
+  e.preventDefault();
+
+  const data = new FormData(form);
+  const nameInput = data.get('name');
+  const positionInput = data.get('position');
+  const officeInput = data.get('office');
+  const ageInput = data.get('age');
+  const salaryInput = data.get('salary');
+  const newSalary = '$' + Number(salaryInput).toLocaleString('en-US');
+
+  if (nameInput.length < 4) {
+    return pushNotification('Error',
+      'Please, enter at least 4 characters', 'error');
+  } else if (positionInput.length <= 0) {
+    return pushNotification('Error',
+      'Please, enter position', 'error');
+  } else if (ageInput < 18 || ageInput > 90) {
+    return pushNotification(
+      'Error', 'Please, enter a valid age', 'error');
+  } else {
+    const row = document.createElement('tr');
```

Comment:
There is no need for `else...if`. The code will not go further after `return` anyway.
Also, the code looks a bit complicated with `else...if`